pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -88,7 +88,7 @@
   private Collection<String> bannedNodeIds;
   private MetricsConfiguration metricsConfiguration;
   private MetricsSystem metricsSystem;
-  private PermissioningConfiguration permissioningConfiguration;
+  private Optional<PermissioningConfiguration> permissioningConfiguration;
```

Comment:
As per earlier comment, initialise this to empty and fill it if the setter is called. This also avoids potential null pointers in build()